content/en/docs/pipelines/installation/overview.md

```diff
@@ -8,15 +8,41 @@ Kubeflow Pipelines offers a few installation options.
 This page describes the options and the features available
 with each option:
 
-* A [standalone](#standalone) Kubeflow Pipelines deployment.
-* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow).
-* **Alpha**: [GCP Hosted ML Pipelines](#marketplace).
+* A [standalone](#standalone) Kubeflow Pipelines deployment is the minimal
+portable installation that only includes Kubeflow Pipelines.
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow) provides
+all Kubeflow components and more integration with each platform.
+* **Beta**: [AI Platform Pipelines](#marketplace) provides easy-to-use management
+UI for installing and using Kubeflow Pipelines on GCP.
+
+## Choosing an installation option
+
+1. Do you want to use other Kubeflow components in addition to Pipelines?
+
+    If yes, choose the [full Kubeflow](#full-kubeflow).
+1. Do you want to use Kubeflow Pipelines with [multi user support](https://github.com/kubeflow/pipelines/issues/1223)?
+
+    If yes, wait for the to be released [full Kubeflow](#full-kubeflow) 1.1.
+1. Do you deploy on Google Cloud Platform (GCP)?
```

Comment:
I think it may make sense to restructure these two top level bullet points to feature KFP standalone, and mention AI Platform Pipelines as a deployment option. For example:

1.  Do you want to deploy Kubeflow Pipelines without the other Kubeflow components? 
  
    If yes, deploy [Kubeflow Pipelines Standalone](#standalone). If you are deploying on Google Cloud, you can also [use AI Platform Pipelines to deploy Kubeflow Pipelines Standalone using a user interface](#marketplace).


This option keeps the focus on KFP Standalone and  removes the duplicate content for users who are deploying on other platforms.